ckan/config/middleware/flask_app.py

```diff
@@ -262,7 +262,7 @@ def make_flask_stack(conf: Union[Config, CKANConfig]) -> CKANApp:
     _register_error_handler(app)
 
     # CSRF
-    app.config['WTF_CSRF_FIELD_NAME'] = "_csrf_token"
```

Comment:
This config declaration has a different default:

https://github.com/ckan/ckan/blob/978875e41643b53fc2c714b2264f32e502db3f8d/ckan/config/config_declaration.yaml#L664-L666

That is probably fine.

But the old name is hardcoded in some JS:

The [meta tag](https://github.com/ckan/ckan/blob/978875e41643b53fc2c714b2264f32e502db3f8d/ckan/templates/base.html#L27) with the token uses the config name, but the code that reads the value [in the js client](https://github.com/ckan/ckan/blob/978875e41643b53fc2c714b2264f32e502db3f8d/ckan/public/base/javascript/client.js#L54) has the name hardcoded.

There's also [this module](https://github.com/ckan/ckan/blob/978875e41643b53fc2c714b2264f32e502db3f8d/ckan/public/base/javascript/modules/confirm-action.js#L106) that uses the old name.

Maybe we can simplify things and enforce a name everywhere (ie ignore this WTForms config option) or alternatively the JS needs to be aware of the field name (proabably adding a separate `csrf_field_name` meta tag that they can read first before getting the actual token value)